### DISCUSSION TITLE
Trying to open ft.Image with src="0.0.0.0:8081:/et/cetera/file.jpg"

### DISCUSSION THREAD

**Discussion by mvwilpe** [flet-dev/flet#4058] [Q&A] [answered]
### Question

Im trying to convert my app to run in docker but run into trouble getting the correct ip-adress.  I have 2 questions:
1. When I specify ft.Image src="http://127.0.0.1:8081:/path/to/file.jpg" it works ok, but "http://0.0.0.0:8081:/.. doesn't work. I've no idea why it doesn't work.
2. I can show a .jpg image, but a .png image isn't visible. Why is that? Do I have to convert it to jpg?

My first try with docker. Obviously I'm doing something wrong here. Anybody have an idea?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by tanmay-bhatgare** [comment]
Any errors or log regarding the issue? Consider using flet debug if none works.

**Comment by mvwilpe** [comment, marked as answer]
Don't bother to reply; just found out myself: Page has .client_ip (the second one about not showing png was mny error.

**Reply by tanmay-bhatgare** [reply]
Great! Mind closing this issue as complete?

**Comment by mvwilpe** [comment]
Done ;)
